components/builder-worker/src/runner/mod.rs

```diff
@@ -461,6 +461,7 @@ impl Runner {
                     util::docker_exporter_spec(&self.workspace),
                     &self.workspace,
                     &self.config.bldr_url,
+                    &self.bldr_token,
```

Comment:
This is _probably_ dangerous. The builder token is a admin token that has permissions to do anything, including getting private packages from repos you're not a member of. You probably want to fetch a token scoped to the user rather than use the builder token.

Reply:
Builds themselves aren't scoped to a user so I'm not certain how we'd do that. Also, this token is stored in memory and used in the workspace anyways today - this is just using it for one more operation.

Reply:
Perhaps we need a new token generation for our workers - something that generates a throwaway token for the build from the origin the package belongs to?

Reply:
This should be fine (for now) - it's no different than what the runner itself is doing in the other steps. The builder tokens themselves are ephemeral and expire in 2 hours.